extensions/redis-client/runtime/src/main/java/io/quarkus/redis/runtime/client/VertxRedisClientFactory.java

```diff
@@ -48,7 +49,8 @@ public static Redis create(String name, Vertx vertx, RedisClientConfig config) {
         if (config.hosts().isPresent()) {
             hosts.addAll(config.hosts().get());
             for (URI uri : config.hosts().get()) {
-                options.addConnectionString(uri.toString().trim());
+                URI withClientName = addClientParameter(name, uri);
```

Comment:
That should be an option (enabled by default).
Also, using the client name makes sense, but should we have a way to pass another value?